org-roam-doctor.el

```diff
@@ -102,7 +102,7 @@ AST is the org-element parse tree."
     (org-element-map ast 'keyword
       (lambda (kw)
         (let ((key (org-element-property :key kw)))
```

Comment:
`org-element` return keys that are always uppercased, so this is incorrect, I think.